docs/examples/sensormanagement/bayesian_search_example_2.py

```diff
@@ -0,0 +1,465 @@
+#
+#                             x[i], y[i] = pol2cart(range_, sensor.dwell_centre[0, 0] \
+#                                                     + sensor.fov_angle / 2 * fov_side) \
+#                                                   + sensor.position[[0, 1], 0]
+#
+#                 data_.append(go.Scatter(x=[x[0], sensor.position[0], x[1]],
+#                                         y=[y[0], sensor.position[1], y[1]]))
+#                 traces_.append(trace_base)
+#
+#                 frame.traces = traces_
+#                 frame.data = data_
+#
+#         # plot radius of sensor
+#         if plot_radius and sensor.max_range != np.inf:
+#
+#             # define the layout
+#             trace_base = len(plt.fig.data)  # number of traces currently in figure
+#             sensor_kwargs = dict(mode='lines', line=dict(dash="dash", color="black"),
+#                                 hoverinfo=None, showlegend=True, name="sensor radius",
+#                                 legendgroup="sensor radius")
+#
+#             plt.fig.add_trace(go.Scatter(sensor_kwargs))  # initialises trace
+#
+#             for frame in plt.fig.frames:
+#
+#                 # get frame time in correct format
+#                 frame_time = datetime.fromisoformat(frame.name)
+#                 traces_ = list(frame.traces)
+#                 data_ = list(frame.data)
+#
+#             for sensor in sensor_history:
+#
+#                 if sensor.timestamp == frame_time:  # if sensor is in current timestep
+#                         circle = GaussianState([[sensor.position[0],
+#                                                 sensor.position[1]]],
+#                                                 np.diag([sensor.max_range ** 2,
+#                                                         sensor.max_range ** 2]))
+#
+#                         points = Plotterly._generate_ellipse_points(circle, [0, 1])
+#
+#                         data_.append(go.Scatter(x=points[0, :],
+#                                                 y=points[1, :]))
+#                         traces_.append(trace_base)
+#
+#                 frame.traces = traces_
+#                 frame.data = data_
+#         return plt
+#
+#
+#     from stonesoup.plotter import AnimatedPlotterly
+#     plt = AnimatedPlotterly(timesteps, width=550)
+#     plt = plot_search_heatmap(plt, x_pos, y_pos, search_cell_info)
+#     plt = plot_moving_sensor(plt, sensor_history, plot_fov=False, plot_radius=True)
+#     plt.fig.update_xaxes(range=[-0.5, 14.5])
+#     plt.fig.update_yaxes(range=[-0.5, 14.5])
+#
+# .. raw:: html
+#
+#     </details>
+#     <br>  
+#     <video autoplay loop controls width=100% height="auto">
+#       <source src="../../_static/bayesian-search-ex2-plt1.mp4" type="video/mp4">
+#     </video>
+#     <br>
+#
+# | 
+#
+# As expected, at each timestep the platform and sensor move to the neighbouring group of cells
+# that has the highest total probability of containing the target.
```

Comment:
Doesn't appear to do that for the first step though. 

I does, however, start from the wrong place in the video. Isn't the initial position (4,4)?